### PR TITLE
en: update error handling descriptions

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

- Add `workaround` for the error system
- Some minor updates

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: #243
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: en/error-handling.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ In the error system, usually, the information of a specific error is as follows:
...
     | `dm-worker`      |  DM-worker service  | `[code=40066:class=dm-worker:scope=internal:level=high] ExecuteDDL timeout, try use query-status to query whether the DDL is still blocking` |
     | `dm-tracer`      |  DM-tracer service  | `[code=42004:class=dm-tracer:scope=internal:level=medium] trace event test.1 not found` |
-    | `schema-tracker` | schema-tracker (during incremental data replication)   | `ErrSchemaTrackerCannotExecDDL,[code=44006:class=schema-tracker:scope=internal:level=high],"cannot track DDL: ALTER TABLE test DROP COLUMN col1"` |
-    | `scheduler`      | Scheduling operations (of data replication tasks)   | `ErrSchedulerNotStarted,[code=46001:class=scheduler:scope=internal:level=high],"the scheduler has not started"` |
+    | `schema-tracker` | schema-tracker (during incremental data replication)   | `[code=44006:class=schema-tracker:scope=internal:level=high],"cannot track DDL: ALTER TABLE test DROP COLUMN col1"` |
+    | `scheduler`      | Scheduling operations (of data replication tasks)   | `[code=46001:class=scheduler:scope=internal:level=high],"the scheduler has not started"` |
```
Comment (TomShawn) [inline en/error-handling.md:45]: [This line](https://github.com/pingcap/docs-dm/pull/243/files#diff-d4fe40beb4439cbd9249eec5df8acf32R45) is missing in the translation.

File: en/error-handling.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ In the error system, usually, the information of a specific error is as follows:
 
     Detailed descriptions of the error. To wrap and store every additional layer of error message on the error call chain, the [errors.Wrap](https://godoc.org/github.com/pkg/errors#hdr-Adding_context_to_an_error) mode is adopted. The message description wrapped at the outermost layer indicates the error in DM and the message description wrapped at the innermost layer indicates the error source.
 
+- `workaround`: Error handling methods.
```
Comment (TomShawn) [inline en/error-handling.md:64]:
```suggestion
- `workaround`: Error handling methods (optional)
```

File: en/error-handling.md [hunk 3]
```diff
@@ -61,6 +61,10 @@ In the error system, usually, the information of a specific error is as follows:
...
+
+    The handling methods for this error. For some confirmed errors (such as configuration errors), DM gives corresponding manual handling methods in `workaround`.
```
Comment (TomShawn) [inline en/error-handling.md:66]:
```suggestion
    The handling methods for this error. For some confirmed errors (such as configuration errors), DM gives the corresponding manual handling methods in `workaround`.
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: en/error-handling.md (hunk 1 above)
Comment (lilin90) [reply en/error-handling.md:45]: Oh, yes!

**Comment by lilin90** [comment]
@TomShawn All updated. PTAL again~ Thanks!

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**